### ISSUE TITLE
Get the n th item in array

### ISSUE THREAD

**Issue by sign0** [dominictarr/JSONStream#163]
Hi !

Do you have a method to retrieve a specific item in an array ?
Example : get the 200'042 th element from an array of 256'000 items.
```
[
    {"header":{...},"data":[......]},
    {"header":{...},"data":[......]},
    ......
    {"header":{...},"data":[......]}, <-- TARGET
    ...
    {"header":{...},"data":[......]},
]
```

Thanks

**Comment by Poetro** [comment]
Since JSONStream gives you a stream of items, you will have to process all, and throw away the unwanted items. Maybe keep a counter of the items processed, so you will know which ones you need to keep.

**Comment by sign0** [comment]
Yes, that's already what I do. But it's very slow in my case and I wondered if it was possible to do otherwise. I will rather try with bash commands to see if it's possible.

Thanks
